app/views/api/_external_site.xml.builder

```diff
@@ -5,7 +5,7 @@ xml.tag!(tag,
 ) do
   xml_string(xml, :name, object.name)
   if !detail
-    xml_minimal_object(xml, :project, Project, object.project_id)
+    xml_minimal_object(xml, :project, :project, object.project_id)
```

Comment:
`... :project, :project ...` looks redundant.
I don'r understand this well enough to understand the intent of `Project` in the original file.

Reply:
I needed to pass in the class before, because I was including the "show" url.  But since I'm not any more, I no longer need the class, so it's easier all around (especially in one particular case associated with comments) to switch to just the "type tag", which is all it needs now.